templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a style="float: right" href="{% url 'taxi:manufacturer-create' %}">+</a>
     </h1>
```

Comment:
Ensure that the '+' link for creating a new manufacturer is correctly aligned and styled. Consider moving the closing `</a>` tag to the next line for better readability and consistency with other HTML elements.